tests/data_model_helpers.py

```diff
@@ -147,24 +147,61 @@ def _get_proto_val_name(field_val) -> str:
     raise RuntimeError(f"Invalid field type specifier: {field_val}")
 
 
-def make_proto_def(message_specs: Dict[str, dict], pkg_suffix: str = None) -> str:
+def make_proto_def(
```

Comment:
Changes in this file support creating temporary data model modules that simulate using precompiled protobufs rather than using `@dataobject`